mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSModelAsNode.kt

```diff
@@ -67,52 +86,73 @@ data class MPSModelAsNode(val model: SModel) : IDeprecatedNodeDefaults {
     override fun getChildren(link: IChildLink): Iterable<INode> {
         return if (link is NullChildLink) {
             emptyList()
-        } else if (link.getUID().endsWith(BuiltinLanguages.MPSRepositoryConcepts.Model.rootNodes.getUID()) ||
-            link.getUID().contains("rootNodes") ||
-            link.getSimpleName() == "rootNodes"
-        ) {
+        } else if (link.conformsTo(builtinModel.rootNodes)) {
             model.rootNodes.map { MPSNode(it) }
+        } else if (link.conformsTo(builtinModel.modelImports)) {
+            ModelImports(model).importedModels.mapNotNull {
+                MPSModelImportAsNode(it.resolve(model.repository), model)
+            }
+        } else if (link.conformsTo(builtinModel.usedLanguages)) {
+            getImportedLanguagesAndDevKits()
         } else {
             emptyList()
         }
```

Comment:
Maybe this could be made more concise with a [table-driven approach](https://medium.com/swlh/replacing-logical-statements-with-table-driven-methods-da1114512134)